modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -477,9 +489,7 @@ If a store copy is required, make a note of the information in the `createStatem
 For example:
 [source,cypher]
 ----
-SHOW INDEXES YIELD type, createStatement
-WHERE type = 'VECTOR'
-RETURN createStatement
+SHOW VECTOR INDEXES YIELD createStatement
```

Comment:
If someone is affected by this issue (<5.12), they cannot use this syntax, right?

Reply:
true, I can switch this one back. I was debating if I wanted to update it or not

Reply:
```suggestion
SHOW INDEXES YIELD type, createStatement
WHERE type = 'VECTOR'
RETURN createStatement
```